### ISSUE TITLE
Terminal Add-on crashes after 15-20 seconds of use

### ISSUE THREAD

**Issue by rudders** [hassio-addons/addon-terminal#18]
# Problem/Motivation

> The terminal add-on allows login and a few commands and then stops functioning and can not be used again until restarted.

## Expected behaviour

> I expected Terminal to continue to operate.

## Actual behaviour

> Terminal loads, interacting with Terminal works for 20 or so seconds (simple things like ls-la, top, uptime commands all works) and then it stops working.

## Steps to reproduce

> Very new a clean implementation of hassio (as of 30th January 2018).

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

log file from restart to crash.
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] 10-hassio: applying... 
[fix-attrs.d] 10-hassio: exited 0.
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Hass.io Add-on: Terminal v2.2.1
 Terminal access to your Home Assistant instance via the web
 From: Community Hass.io Add-ons
 By: Franck Nijhof <frenck@addons.community>
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to INFO
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] 02-updates.sh: executing... 
INFO: You are running the latest version of this add-on
[cont-init.d] 02-updates.sh: exited 0.
[cont-init.d] 03-version-requirements.sh: executing... 
INFO: Supervisor version requirements checks passed.
[cont-init.d] 03-version-requirements.sh: exited 0.
[cont-init.d] 10-requirements.sh: executing... 
[cont-init.d] 10-requirements.sh: exited 0.
[cont-init.d] 30-link-common-dirs.sh: executing... 
[cont-init.d] 30-link-common-dirs.sh: exited 0.
[cont-init.d] 40-zsh.sh: executing... 
[cont-init.d] 40-zsh.sh: exited 0.
[cont-init.d] 50-user-ssh-folder.sh: executing... 
[cont-init.d] 50-user-ssh-folder.sh: exited 0.
[cont-init.d] 80-user-packages.sh: executing... 
[cont-init.d] 80-user-packages.sh: exited 0.
[cont-init.d] 81-user-scripts.sh: executing... 
[cont-init.d] 81-user-scripts.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
starting version 3.2.4
[services.d] done.
INFO: Starting the ttyd daemon
INFO: Starting the ttyd daemon
[2018/02/01 10:49:06:8219] ERR: ERROR on binding fd 6 to port 7681 (-1 98)
[2018/02/01 10:49:06:8220] ERR: init server failed
[2018/02/01 10:49:06:8220] ERR: Failed to create default vhost
[2018/02/01 10:49:06:8220] ERR: libwebsockets init failed
```

**Comment by cetken** [comment]
I'm having the exact same issue. Even after upgrading to the latest version (0.62.1) the problem persists.

**Comment by frenck** [comment]
I'm unable to reproduce this issue, nevertheless, this one has been reported a couple of times before. 

Could you guy provide more information about the setup?
Like hardware, port settings, add-on settings, SSL stuff, proxies?

Without a true reproduction, I'm kinda lost... 😢

**Comment by frenck** [comment]
I've just released v2.3.0 of the add-on, which ships with a lot of new dependencies, please check if that solves this issue.

**Comment by rudders** [comment]
Problem resolved for me as of 2.3.0. Thanks!

**Comment by frenck** [comment]
Great! Thanks for letting me know!
